tests/snuba/api/endpoints/test_organization_events_trace.py

```diff
@@ -555,6 +555,128 @@ def test_sibling_transactions(self):
         for child in gen2_parent["children"]:
             assert child["event_id"] in gen3_event_siblings
 
+    def test_with_orphan_siblings(self):
+        parent_span_id = uuid4().hex[:16]
+        root_event = self.create_event(
+            trace=self.trace_id,
+            transaction="/orphan/root",
+            spans=[],
+            # Some random id so its separated from the rest of the trace
+            parent_span_id=parent_span_id,
+            project_id=self.project.id,
+            duration=1000,
+        )
+        root_sibling_event = self.create_event(
+            trace=self.trace_id,
+            transaction="/orphan/root-sibling",
+            spans=[],
+            parent_span_id=parent_span_id,
+            project_id=self.project.id,
+            duration=1000,
+        )
+
+        with self.feature(self.FEATURES):
+            response = self.client.get(
+                self.url,
+                data={"project": -1},
+                format="json",
+            )
+
+        assert response.status_code == 200, response.content
+        import pprint
+
+        pprint.pprint(response.data)
```

Comment:
Was this left by accident?

Reply:
.... hundo p, i guess the lint doesn't catch pprint